data-pipeline/de-duplication/src/main/java/org/ekstep/ep/samza/service/DeDuplicationService.java

```diff
@@ -36,8 +36,8 @@ public void process(DeDuplicationSource source, DeDuplicationSink sink) throws E
 				sink.toSuccessTopic(event);
 				return;
 			}
-			if (!((null != event.producerId() && config.producerId().contains(event.producerId()))
-					|| config.serverEventEid().contains(event.eid()))) {
+
+			if (isDupCheckRequired(event)) {
```

Comment:
Default behaviour should not break. If the inclusion/exclusion filter is not provided, all the events should be deduped.